homeassistant/components/vizio/config_flow.py

```diff
@@ -108,11 +118,11 @@ def __init__(self) -> None:
 
             # Check if new config entry matches any existing config entries
             for entry in self.hass.config_entries.async_entries(DOMAIN):
-                if entry.data[CONF_HOST] == user_input[CONF_HOST]:
+                if _host_is_same(entry.data[CONF_HOST], user_input[CONF_HOST]):
                     errors[CONF_HOST] = "host_exists"
                     break
 
-                if entry.data[CONF_NAME] == user_input[CONF_NAME]:
+                if _name_is_same(entry.data[CONF_NAME], user_input[CONF_NAME]):
```

Comment:
This one seems a bit overkill.

Reply:
Yes I was originally planning to change the name check logic as well to cover more cases but decided against it. I will revert this.